dev/bots/test.dart

```diff
@@ -85,6 +85,22 @@ const List<String> kWebTestFileKnownFailures = <String>[
 const String kSmokeTestShardName = 'smoke_tests';
 const List<String> _kAllBuildModes = <String>['debug', 'profile', 'release'];
 
+// The seed used to shuffle tests.  If not passed with
+// --test-randomize-ordering-seed=<seed> on the command line, it will be set the
+// first time it is accessed. Pass zero to turn off shuffling.
+String? _shuffleSeed;
+String get shuffleSeed {
+  if (_shuffleSeed == null) {
+    // Change the seed at 7am, UTC.
```

Comment:
Why 7am UTC? Just curious.

Reply:
It is approximately midnight Pacific, and for the majority of Flutter framework developers will switch sometime between midnight and 9am. I would have made it be midnight Pacific, except that Dart has no way to compute whether it is daylight savings time or not, so it is sometimes 11pm, sometimes midnight. I also didn't want to make it midnight local time, since who knows where our servers will be running the tests in the future, so UTC was a better bet.